tests/manager/searchCustomer/managerCanSearchCustomerByPostalCode.spec.js

```diff
@@ -19,7 +21,17 @@ test.beforeEach( async ({ page }) => {
   lastName = faker.person.lastName();
   postalCode = faker.location.zipCode(); 
 
+  firstName = faker.person.firstName();
+  lastName = faker.person.lastName();
+  postalCode = faker.location.zipCode(); 
```

Comment:
The variables `firstName`, `lastName`, and `postalCode` are initialized twice. The second initialization is redundant and should be removed to clean up the code.